app/src/main/java/lt/vilnius/tvarkau/utils/BitmapUtils.java

```diff
@@ -0,0 +1,61 @@
+package lt.vilnius.tvarkau.utils;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+
+import java.io.File;
+
+public class BitmapUtils {
+
+    public static Bitmap createBitmap(Uri uri) {
+
+        final int requiredWidth = 1600;
+        final int requiredHeight = 1600;
+
+        BitmapFactory.Options options = new BitmapFactory.Options();
+        options.inJustDecodeBounds = true;
+        BitmapFactory.decodeFile(new File(uri.getPath()).getAbsolutePath(), options);
+        options.inSampleSize = calculateInSampleSize(options, requiredWidth, requiredHeight);
+        options.inJustDecodeBounds = false;
+        Bitmap bitmap = BitmapFactory.decodeFile(new File(uri.getPath()).getAbsolutePath(), options);
```

Comment:
`uri.getPath()).getAbsolutePath()` could be written only once (create a variable and store returned value) in this method. .